sass/directives/_accents.scss

```diff
@@ -47,3 +47,54 @@
   width: 100%;
   left: 0;
 }
+
+@mixin return-to-top {
```

Comment:
This PR lacks the use of any variables - do you think there's any opportunity for re-using of existing variables, or creating a new set of variables for the `return-to-top` component?